tklamq_lib/amqp.py

```diff
@@ -156,11 +165,10 @@ def decode_message(message_data, secret=None):
     """
     sender = str(message_data['sender'])
     content = base64.urlsafe_b64decode(str(message_data['content']))
-    timestamp = datetime(*map(lambda f: int(f), message_data['timestamp-utc']))
+    timestamp = datetime(*[int(f) for f in message_data['timestamp-utc']])
```

Comment:
You can actually get even more simplified by using `map` but ditching the lambda (as the lambda was doing nothing other than calling int, so we can just use int directly) `datetime(*map(int, message_data['timestamp-utc']))`

Reply:
Ah ok, thanks. FWIW I think that this is another example of `2to3` giving code that works, but isn't quite ideal...